server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java

```diff
@@ -129,11 +129,12 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     }
 
     private void addContentDispositionHeader(Optional<String> optionalName, HttpServletResponse resp) {
-        optionalName.ifPresent(name -> resp.addHeader("Content-Disposition", "attachment; filename=\"" + name + "\""));
```

Comment:
Then maybe we can keep it if it is composed only of ascii characters... I mean it is an important information that directly affect user display and can convey some usefull information.
If it is possible to keep this information when we send a mail, then we shoud do it...